src/components/project/hero.tsx

```diff
@@ -1,5 +1,259 @@
 import Image from "next/image";
 import { IProjectData, Project } from "../../interfaces/project";
+import { colors, dimensions, breakpoints } from "../../styles/variables";
+import styled from "@emotion/styled";
+import { getRemSize } from "../../styles/globalCss";
+import { Row } from "../global/grid/Row";
+import { Col } from "../global/grid/Col";
+import ArrowUpRight from "../../icons/arrowUpRight";
+import useIsDesktop from "../../hooks/useIsDesktop";
+
+interface IStyledDivImage {
+  imageSrc: string;
+}
+
+const StyledLogoImage = styled.div<{ dark: boolean }>`
+  position: absolute;
+  top: 14px;
+  left: 8px;
+  z-index: 1001;
+  color: ${(props) => (props.dark ? colors.white : colors.black)};
+  img {
+    fill: currentColor;
+  }
+  // @media (max-width: ${breakpoints.md}px) {
+  //   display: none;
+  // }
+`;
+
+const StyledDivImage = styled.div<IStyledDivImage>`
+  overflow: hidden;
+  position: absolute;
+  background-image: url(${(props) => props.imageSrc});
+  background-size: cover;
+  background-position: center;
+  width: 100%;
+  height: 585px;
+  left: 0;
+  top: 0;
+  z-index: -1;
+  border-radius: 0px 0px 20px 20px;
+
+  @media (max-width: ${breakpoints.md}px) {
+    height: 650px;
+  }
+`;
+
+const StyledHeading1 = styled.h1`
+  font-size: ${getRemSize(dimensions.headingSizes.display2.mobile)};
+  font-weight: 400;
+  letter-spacing: 6px;
+  color: ${colors.white};
+
+  @media all and (max-width: 1100px) {
+    font-size: ${getRemSize(dimensions.headingSizes.medium.desktop)};
+  }
+  @media (max-width: ${breakpoints.md}px) {
+    font-size: ${getRemSize(dimensions.headingSizes.display2.mobile)};
+    font-weight: 500;
+    letter-spacing: 2px;
+    line-height: 1;
+    margin: 0;
+  }
+`;
+
+const StyledHeading2 = styled.h2`
+  font-size: ${getRemSize(dimensions.headingSizes.cta.mobile)};
+  font-weight: 400;
+  letter-spacing: 6px;
+  color: ${colors.white};
+
+  @media all and (max-width: 1100px) {
+    font-size: ${getRemSize(dimensions.headingSizes.cta.mobile)};
+    letter-spacing: 3px;
+  }
+  @media all and (max-width: 1001px) {
+    font-size: ${getRemSize(dimensions.headingSizes.cta.mobile)};
+    letter-spacing: 1px;
+  }
+  @media (max-width: ${breakpoints.md}px) {
+    font-size: ${getRemSize(dimensions.headingSizes.cta.desktop)};
+    letter-spacing: 2px;
+    line-height: 0;
+    margin: 0;
```

Comment:
`line-height: 0;` messes up on smaller screens, instead use margin-top negative value:

```suggestion
    line-height: 0;
    margin: -20px 0 0 0;
```